src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -2587,31 +2419,29 @@ public void T131_String_OpenOrCreate_ReadWrite_Star()
                   .GetRelationshipsByType(DocumentRelationshipType)
                   .FirstOrDefault();
 
-                if (docPackageRelationship != null)
-                {
-                    Uri documentUri =
-                        PackUriHelper
-                        .ResolvePartUri(
-                           new Uri("/", UriKind.Relative),
-                                 docPackageRelationship.TargetUri);
-
-                    var mainPart = package.GetPart(documentUri);
+                Assert.NotNull(docPackageRelationship);
+                Uri documentUri =
+                    PackUriHelper
+                    .ResolvePartUri(
+                       new Uri("/", UriKind.Relative),
+                             docPackageRelationship.TargetUri);
 
+                var mainPart = package.GetPart(documentUri);
 
-                    using (Stream partStream = mainPart.GetStream(FileMode.Open, FileAccess.Read))
-                    {
-                        XDocument xd = XDocument.Load(partStream);
-                        Assert.True(xd.DescendantNodes().Count() != 0);
-                    }
+                using (Stream partStream = mainPart.GetStream(FileMode.Open, FileAccess.Read))
+                {
+                    XDocument xd = XDocument.Load(partStream);
+                    Assert.True(xd.DescendantNodes().Count() != 0);
```

Comment:
Probably missed because I was just going through looking for explicit `Assert.True(false...` and `Assert.False(true...`, and ends up showing in the diff because I switched to checking for null as a test, instead of as a condition (so indentation changed).  I can fix these, but we probably need to audit most tests for stuff like this.